bamboo/lint.sh

```diff
@@ -4,5 +4,4 @@ set -ex
 
 npm install
 npm run bootstrap-no-build
-npm run lint-md
```

Comment:
To clarify, I

1. Renamed the `npm run lint` command to `npm run lint-js`
2. Updated the `npm run lint` command to run both `lint-js` and `lint-md`